.github/workflows/flutter.yaml

```
@@ -0,0 +1,36 @@
+      - run: flutter analyze
+
+      - name: Build Linux
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y ninja-build libgtk-3-dev
+          flutter build linux --release
+      - name: Upload Linux Artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: release-linux
+          path: build/linux/x64/release/bundle
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)